controller/AbstractRestQti.php

```diff
@@ -67,25 +72,25 @@ abstract class AbstractRestQti extends \tao_actions_RestController
      *
      * @return string
      */
-    abstract protected function getTaskName();
+    abstract protected function getTaskName(): string;
```

Comment:
I'v checked just this one  and its child \taoQtiTest_actions_RestQtiTests::getTaskName has a problem with new return type. Probably we need to check other methods also
![image](https://github.com/user-attachments/assets/7caec2c3-8259-4da7-8727-18fdaa957ca2)

Reply:
I will revert it back for now